builder/base/kss_builder_base.js

```diff
@@ -427,7 +427,7 @@ class KssBuilderBase {
               });
               /* eslint-enable no-loop-func */
             } else if (typeof this.options[key] === 'string') {
-              this.options[key] = path.resolve(this.options[key]);
+              this.options[key] = path.resolve(process.cwd(), this.options[key]);
```

Comment:
Is this `process.cwd()` suppose to be here?

Reply:
I think path should be relative to working directory (or config file) no ? where it should be ?

Reply:
ping @RobLoach

Reply:
Unsure where it should be based from... process.cwd() seems fine?

Reply:
After some test and I need to improve it.
Path in `source` or `homepage` options are not correctly resolved if it's from config file or command line.

Reply:
I think it's fixed now